pkgs/top-level/aliases.nix

```diff
@@ -1187,7 +1187,19 @@ mapAliases {
 
   lixVersions = lixPackageSets.renamedDeprecatedLixVersions; # Added 2025-03-20, warning in ../tools/package-management/lix/default.nix
 
-  llvmPackages_git = (callPackages ../development/compilers/llvm { }).git;
+  inherit
+    (rec {
+      llvmPackagesSet = callPackages ../development/compilers/llvm { };
```

Comment:
A `let … in` would probably be clearer than `rec`, and then move `llvmPackages_git` out of the `inherit`?